### ISSUE TITLE
TypeError: the JSON object must be str, bytes or bytearray, not list

### ISSUE THREAD

**Issue by banescusebi** [crytic/crytic-compile#178]
Hey, I'm getting the following error when trying to compile any Solidity file on macOS BigSur using `crytic-compile`:

```
Traceback (most recent call last):
  File "/usr/local/bin/crytic-compile", line 33, in <module>
    sys.exit(load_entry_point('crytic-compile==0.1.12', 'console_scripts', 'crytic-compile')())
  File "/usr/local/lib/python3.9/site-packages/crytic_compile-0.1.12-py3.9.egg/crytic_compile/__main__.py", line 157, in main
    compilations = compile_all(**vars(args))
  File "/usr/local/lib/python3.9/site-packages/crytic_compile-0.1.12-py3.9.egg/crytic_compile/crytic_compile.py", line 1076, in compile_all
    compilations.append(CryticCompile(target, **kwargs))
  File "/usr/local/lib/python3.9/site-packages/crytic_compile-0.1.12-py3.9.egg/crytic_compile/crytic_compile.py", line 137, in __init__
    self._compile(**kwargs)
  File "/usr/local/lib/python3.9/site-packages/crytic_compile-0.1.12-py3.9.egg/crytic_compile/crytic_compile.py", line 987, in _compile
    self._platform.compile(self, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/crytic_compile-0.1.12-py3.9.egg/crytic_compile/platform/solc.py", line 180, in compile
    crytic_compile.abis[contract_name] = json.loads(info["abi"])
  File "/usr/local/Cellar/python@3.9/3.9.1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 339, in loads
    raise TypeError(f'the JSON object must be str, bytes or bytearray, '
TypeError: the JSON object must be str, bytes or bytearray, not list
```

FYI: I use `solc-select` and have selected the right `solc` version and also successfully compiled the same contract using the `solc contract.sol` command. Any ideas of why this error is showing up?

**Comment by montyly** [comment]
Hi @banescusebi. Thank you for reporting this.

Can you try with [crytic-compile@0.2.0](https://github.com/crytic/crytic-compile/releases/tag/0.2.0)? We fixed a couple of issues related to solc 0.8.

**Comment by banescusebi** [comment]
Hi @montyly, I confirm that it works with crytic-compile@0.2.0

Thanks for your help.
